### PR TITLE
Add Support for GE Water Filters

### PR DESCRIPTION
**Pull request by bendavis**
Add initial support for GE Water Filters. 

Right now added sensors for mode of the filter, filter life, and addition to flow rate & day usage.

HA Integration:
wh_filter_ha

### CODE REVIEW

File: custom_components/ge_home/update_coordinator.py [hunk 1]
```diff
@@ -270,31 +293,35 @@ def all_appliances_updated(self) -> bool:
...
         """When we get an appliance list, mark it and maybe trigger all ready."""
-        _LOGGER.debug('Got roster update')
+        _LOGGER.debug("Got roster update")
         self.last_update_success = True
         if not self._got_roster:
             self._got_roster = True
-            #TODO: Probably should have a better way of confirming we're good to go...
-            await asyncio.sleep(5)  # After the initial roster update, wait a bit and hit go
+            # TODO: Probably should have a better way of confirming we're good to go...
+            await asyncio.sleep(
```
Comment (simbaja) [inline custom_components/ge_home/update_coordinator.py:301]: I'm seeing things like this throughout - guessing you're using a code formatter.  Can you update it to not perform this particular type of reformat?

File: custom_components/ge_home/entities/waterfilter/filter_life_remaining.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+from gehomesdk import ErdCode, ErdOperatingMode
+from gehomesdk.erd.values.common.erd_measurement_units import ErdMeasurementUnits
+from typing import Optional
+
+from ..common import GeErdSensor
+
+
+class ErdFilterLifeRemainingSensor(GeErdSensor):
```
Comment (simbaja) [inline custom_components/ge_home/entities/waterfilter/filter_life_remaining.py:8]: just to stick with conventions, can you prefix the new entity types with Ge?

**Review by simbaja — CHANGES_REQUESTED** [review]

**Comment by simbaja** [comment]
Great job adding this support and thanks for the contribution!!  Just a few minor changes requested and we can get this merged into the code.

**Comment by bendavis** [comment]
I make these changes - I also noticed ( when loading on my main instance with HACS) that some of the imports are wrong. I'll push that up too.

**Comment by simbaja** [comment]
Any updates on the changes?  You mentioned adding some additional bugfixes so didn't want to just push this over and fix up the couple of comments.

**Pull request merged**